dallinger/frontend/templates/base/layout.html

```diff
@@ -17,12 +17,12 @@
         <div id="spinner" />
         </div>
         {% block libs %}
-            <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
+            <script src="/static/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
```

Comment:
The other scripts we already ship with are in `/static/scripts/`. Seems like these should go in the same place?

Reply:
Indeed! I moved both `jquery` and `fingerprintjs2` to the `scripts/` directory.